tools/generators/migrate-converged-pkg/index.ts

```diff
@@ -758,6 +760,7 @@ function updateApiExtractor(tree: Tree, options: NormalizedSchemaWithTsConfigs)
   writeJson(tree, joinPathFragments(options.paths.configRoot, 'api-extractor.json'), apiExtractor.main);
 
   updateJson(tree, options.paths.packageJson, (json: PackageJson) => {
+    json.scripts = json.scripts ?? {};
```

Comment:
ts now strictly checks undefined which cannot be used within `Object.assign` context